dask/dataframe/tests/test_arithmetics_reduction.py

```diff
@@ -1363,7 +1376,7 @@ def test_reductions_frame_dtypes_numeric_only():
         assert_eq(
             getattr(df, func)(**kwargs),
             getattr(ddf, func)(**kwargs),
-            check_dtype=func in ["mean", "max"],
+            check_dtype=func in ["mean"],
```

Comment:
I was curious if this was still needed, so I removed it and locally things passed. Including a small change to remove the special `check_dtype` handling here -- let's just use the default of always checking.

Reply:
Yeah I think this makes sense. Didn't pay too much attention, since I expect that we get rid of this test soonish